app/main/views/suppliers.py

```diff
@@ -88,9 +88,12 @@ def dashboard():
 
 
 @main.route('/edit', methods=['GET'])
+@main.route('/edit/<path:step>', methods=['GET'])
+@main.route('/edit/<path:step>/<path:substep>', methods=['GET'])
 @login_required
-def edit_supplier(supplier=None, errors=None):
-    if not supplier:
+def supplier_edit(step=None, substep=None):
+
+    if feature.is_active('SELLER_EDIT'):
```

Comment:
why wrap the entire function in the if?  will the new "edit" be the only caller of this route?